app/.env

```diff
@@ -0,0 +1,3 @@
+REACT_NATIVE_TMDB_TOKEN=eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiIwZjdkYWM5ZTMyYTFkNWY3NmE5NDkzN2FhNzA3MGU5OCIsInN1YiI6IjU5MGIxYTgzOTI1MTQxNGY4ZDAxNDVmNCIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.455dKo4VnCcefIRyyjGwUr8sc2wctKUAQ-26jemtFtw
```

Comment:
This should never be committed but it has as an exception because this is a tech test so it will allow you to test the app. 
These keys will be deactivated after this PR is reviewed.